src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -793,7 +796,9 @@ def _stringify_generator_output(self, outputs: dict):
 
     def _submit_to_scheduler(self, context: FlowExecutionContext, inputs, nodes: List[Node]) -> Tuple[dict, dict]:
         if not isinstance(self._node_concurrency, int):
-            raise NodeConcurrencyNotFound("Need to set node concurrency as using flow executor.")
+            raise NodeConcurrencyNotFound(
+                message_format=f"Need to set valid node concurrency to run, current value is {self._node_concurrency}."
```

Comment:
Is it good to format the {self._node_concurrency} into the message?

Reply:
Usually it's set by `exec_bulk`, `exec_line`.
The content will be changed, but we may need it to investigate.
Although I'm not sure its content will never be customer content.